src/widgets/Header.tsx

```diff
@@ -0,0 +1,97 @@
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
+import Resize, { ContentRect } from '@dojo/framework/widget-core/meta/Resize';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import Link from '@dojo/framework/routing/ActiveLink';
+
+const logo = require('../assets/logo.svg');
+
+import * as css from './Header.m.css';
+
+const pages = ['Blog', 'Documentation', 'Examples', 'Playground', 'Community'];
+
+export default class Menu extends WidgetBase {
+	protected expanded = false;
+
+	protected get isSmall(): boolean {
```

Comment:
Is there any reason why we can't call the meta in the render? I thought `isSmall` was a standard property so was confused where the resize observer was being called! Perhaps it's a bit unexpected to for side effects to be called in getters.

Reply:
Calling `this.meta` causes side effects? I made this a getter because I was originally calling it in multiple places and didn't want to duplicate the code, but I can put it in render now.

Reply:
Sorry I meant as in actually putting logic in a getter, rather than the `this.meta` causing side effects. Bad choice of words from me there.